django/conf/global_settings.py

```diff
@@ -534,6 +534,8 @@
     'django.contrib.auth.hashers.CryptPasswordHasher',
 ]
 
+PASSWORD_VALIDATORS = []
```

Comment:
Since this is part of a contrib application I think this should be named `AUTH_PASSWORD_VALIDATORS`.